aten/src/ATen/native/native_functions.yaml

```diff
@@ -6798,39 +6798,39 @@
     SparseCsrCPU, SparseCsrCUDA: sparse_compressed_to_sparse
   autogen: to_sparse.sparse_dim_out
 
-- func: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None) -> Tensor
+- func: to_sparse(Tensor self, *, Layout? layout=None, int[2]? blocksize=None, int? n_dense_dim=None) -> Tensor
```

Comment:
Can we change this to use `dense_dim` instead of `n_dense_dim`? We also don't use `n_dim`, but `dim` for arguments and this is a similar property. Of course happy to be convinced otherwise if there's strong arguments against that.

Reply:
I've been pondering the same - changed, thanks.

Reply:
no objections